airflow/providers/docker/operators/docker.py

```diff
@@ -289,6 +279,18 @@ def execute(self, context):
 
         return self._run_image()
 
+    def _set_cli(self):
+        tls_config = self.__get_tls_config()
+
+        if self.docker_conn_id:
+            self.cli = self.get_hook().get_conn()
+        else:
+            self.cli = APIClient(
```

Comment:
```suggestion
    def _set_cli(self):
        if self.docker_conn_id:
            self.cli = self.get_hook().get_conn()
        else:
            tls_config = self.__get_tls_config()
            self.cli = APIClient(
```
The first case is easier to understand when reading the code.

Reply:
I was just copy pasting this part. You are right, this is nicer :)